subnets.html

```diff
@@ -374,9 +437,10 @@
 {
   preloadSubnetImages();
   args = parseQueryString();
-  if (args['network'] && args['mask'] && args['division']) {
+  if (args['network'] && args['mask'] && args['division'] && args['comments']) {
     document.forms['calc'].elements['network'].value = args['network'];
     document.forms['calc'].elements['netbits'].value = args['mask'];
+    curComments = JSON.parse(args['comments']);
```

Comment:
Oh shoot, this breaks all old links. We should probably just use comments if provided. 

```suggestion
   if (args['network'] && args['mask'] && args['division']) {
    document.forms['calc'].elements['network'].value = args['network'];
    document.forms['calc'].elements['netbits'].value = args['mask'];

    if (args['comments']) {
      curComments = JSON.parse(args['comments']);
    } else {
      curComments = {};
    }
```